Kconfig.sof

```diff
@@ -122,6 +122,14 @@ config COMPILER_INLINE_FUNCTION_OPTION
         help
           When enabled, -fno-inline-function option is not passed to compiler
 
+config COLD_STORE_EXECUTE_DRAM
```

Comment:
I like this concept of the cold store. i.e. we keeps things in the cold store until we need to use them.

Reply:
@lgirdwood rather "unless," than "until." In most cases e.g. with "cold base firmware code" it's never copied to SRAM, it's marked as non-performance critical and is always executed in DRAM. Same with LLEXT module code. By default the same is done with data. But with data we plan an optimisation where one can explicitly request a copy of "cold" data in SRAM for a limited period of time.